### PR TITLE
Feature/gocart userguide

### LINKED ISSUE #1186: UFS documentation

### PR DESCRIPTION
**Pull request by lipan-NOAA**
# PR Checklist

- [x] This PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [ ] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below.

- [ ] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

Provide a detailed description of what this PR does. What bug does it fix, or what feature does it add? Is a change of answers expected from this PR? Are any library updates included in this PR (modulefiles etc.)?

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes #<[795](https://github.com/ufs-community/ufs-weather-model/issues/795)>

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [ ] hera.intel
- [ ] hera.gnu
- [ ] orion.intel
- [ ] cheyenne.intel 
- [ ] cheyenne.gnu
- [ ] gaea.intel 
- [ ] jet.intel
- [ ] wcoss_cray
- [ ] wcoss_dell_p3
- [ ] opnReqTest for newly added/changed feature
- [ ] CI

## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

Do PRs in upstream repositories need to be merged first? No

### CODE REVIEW

**Comment by junwang-noaa** [comment]
@lipan-NOAA Thanks for adding the documentation. Would you please also add S2SAW in section 4.2.4  "nems.configure file"? You can see the nems.configure file in the coupld_bmark_p8 RT test run directory. Thanks.

**Comment by lipan-NOAA** [comment]
@junwang-noaa I will add it. Thanks

**Comment by junwang-noaa** [comment]
@lipan-NOAA would you please also turn on the atm_aero test on all the platforms? Please remove "+ hera.intel" in the following lines in tests/rt.conf

```
COMPILE | -DAPP=ATMAERO -DCCPP_SUITES=FV3_GFS_v16                                                                               | + hera.intel                              | fv3 |
RUN     | control_atm_aerosols                                                                                                  | + hera.intel                              | fv3 |
```

File: doc/UsersGuide/source/InputsOutputs.rst [hunk 1]
```diff
@@ -738,41 +738,104 @@ GOCART
 Static datasets (i.e., *fix files*)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. TODO:: GOCART information in progress
+The static input files for GOCART configurations are listed and described in :numref:`Table %s <GOCART_ControlFiles>`.
 
-The static input files for global configurations are listed and described in :numref:`Table %s <GOCART_FixFiles>`.
+.. _GOCART_ControFiles:
```
Comment (DeniseWorthen) [inline doc/UsersGuide/source/InputsOutputs.rst]: This is a typo (Control, not Contro).

**Comment by DeniseWorthen** [comment]
Could you also please update sections 2.1 which documents the official repo and section 2.2 which documents the directory layout for GOCART?

File: doc/UsersGuide/source/InputsOutputs.rst [hunk 2]
```diff
@@ -738,41 +738,104 @@ GOCART
...
-.. TODO:: GOCART information in progress
-
-The input files containing grid information and the initial conditions for global configurations are listed and described in :numref:`Table %s <GOCART_GridICFiles>`.
-
-.. _GOCART_GridICFiles:
-
-.. list-table:: *Input files containing grid information and initial conditions for global configurations*
-   :widths: 35 50 15
-   :header-rows: 1
+Running GOCART in UFS does not require aerosol initial conditions, as aerosol models can always start from scratch (cold start), however, this approach does require more than two weeks of model spin-up to obtain reasonable aerosol simulation results. Therefore, the most popular method is to take previous aerosol simulation results. The result is not necessarily from the same model, it could be from a climatology result, such as MERAA2, or a different model but with the same aerosol species and bin/size distribution.
```
Comment (DeniseWorthen) [inline doc/UsersGuide/source/InputsOutputs.rst]: I would suggest you break the first line into two sentences:
```
Running GOCART in UFS does not require aerosol initial conditions, as aerosol models can always start from scratch (cold start). However,.....
```

File: doc/UsersGuide/source/InputsOutputs.rst [hunk 3]
```diff
@@ -738,41 +738,104 @@ GOCART
...
 
-   * - Filename
-     - Description
-     - Date-dependent
-   * -
-     -
-     -
+The aerosol initial input currently read by GOCART is the same format as the UFSAtm initial input data format of "gfs_data_tile[1-6].nc" in :numref:`Table %s <GridICFiles>`, so the aerosol initial conditions should be combined with the meteorological initial conditions as one initial input file. There are many tools available for this purpose. The global workflow that supports UFS models always provides a solution for this.
```
Comment (DeniseWorthen) [inline doc/UsersGuide/source/InputsOutputs.rst]: Are you referring to ufs-utils here?

File: doc/UsersGuide/source/InputsOutputs.rst (hunk 1 above)
Comment (lipan-NOAA) [reply doc/UsersGuide/source/InputsOutputs.rst]: corrected it

File: doc/UsersGuide/source/InputsOutputs.rst (hunk 2 above)
Comment (lipan-NOAA) [reply doc/UsersGuide/source/InputsOutputs.rst]: changed it

File: doc/UsersGuide/source/InputsOutputs.rst (hunk 3 above)
Comment (lipan-NOAA) [reply doc/UsersGuide/source/InputsOutputs.rst]: yes, added it

**Comment by lipan-NOAA** [comment]
@DeniseWorthen sections 2.1 and 2.2 were updated.

**Review by DeniseWorthen — APPROVED** [review]

**Comment by DeniseWorthen** [comment]
Combined with PR #1189

**Pull request closed without merging**